platforms/platform-alexa/src/AlexaUser.ts

```diff
@@ -24,6 +24,10 @@ export class AlexaUser extends JovoUser<Alexa> {
     return this.jovo.$request.session?.user?.userId || 'AlexaUser';
   }
 
+  get accessToken(): string | undefined {
+    return this.jovo.$request.session?.user.accessToken;
```

Comment:
If `this.jovo.$request.session` is undefined, then `user` will be undefined too, probably resulting in an error such as this one:
```typescript
TypeError: Cannot read property 'accessToken'  of undefined
```
This can be solved by adding a question mark to `user` as well.

Reply:
In my understanding. If `this.jovo.$request.session` is undefined, it instantly returns `undefined`. `user` is irrelevant for an undefined `$session`.

```typescript
interface FooBar {
  foo?: {
    bar: string;
  };
}

const obj: FooBar = {};

console.log(obj.foo?.bar);

```

Result:
```
undefined
```

Reply:
Sorry, I messed my initial comment up. What I meant is that if `this.jovo.$request.session` is defined, but `user` is not, then it will throw said error. We check for this exact thing in the getter for `AlexaUser.prototype.id` above, that's why I mentioned it. If we can safely assume for `user` to be always defined, this conversation can be resolved.

Reply:
https://github.com/alexa/alexa-apis-for-nodejs/blob/009292ae927b7ac587d9eeb108a7134930d9a48c/ask-sdk-model/index.ts#L668

`user` is not optional.